api/models/people.py

```diff
@@ -121,50 +153,94 @@ class Student(models.Model):
         verbose_name="CV help status",
         help_text="Does the student need help with CV at the moment?",
     )
-    status = models.ForeignKey(
-        StudentStatus,
-        on_delete=models.PROTECT,
-        verbose_name="Group studies status",
-        help_text="Status of a student with regard to group studies",
+
+    # JSONField because this will come from external API, so it's good to be protected from changes
+    # Just a reminder: the written test is a model with ForeignKey to Student, no field needed here
+    smalltalk_test_result = models.JSONField(
+        blank=True, null=True, help_text="JSON received from SmallTalk API"
     )
 
     # The general rule is that one student can only learn one language,
     # but we don't want to limit this in the database.
     teaching_languages_and_levels = models.ManyToManyField(TeachingLanguageAndLevel)
```

Comment:
well languages being learnt and languages being taught have to come from the same set of languages :) I call them `TeachingLanguage` :) I could just put `Language` of course. It's just that at first the owners wanted to store the native language as well, so I had `NativeLanguage` and `TeachingLanguage`. Now only `TeachingLanguage` is left. But I don't think it's that confusing to be honest :)

Reply:
As you wish, really not very important, but I think in future it could be confusing.
 
But I think, there are nothing bad to call it `teaching_languages` in `Teacher` model and `studying_languages` (or taught) in `Student` model. Even if it's M2M to `TeachingLanguageAndLevel`. And also I don't understand why we can't rename model to just `Language` if it's only one left :)

Reply:
yeah, let's just rename it to `Language`.